plugins/openpgp-api-lib/openpgp-api/src/main/java/org/openintents/openpgp/util/OpenPgpKeyPreference.java

```diff
@@ -73,6 +73,15 @@ public void setDefaultUserId(String userId) {
 
     @Override
     protected void onClick() {
+        bindServiceAndGetSignKeyId(new Intent());
+    }
+
+    private void bindServiceAndGetSignKeyId(final Intent data) {
+        if (mServiceConnection != null) {
```

Comment:
maybe throw in `&& mServiceConnection.isBound()`, there are cases where the connection might die